backend/tests/attendanceStatistics.test.ts

```diff
@@ -662,18 +662,6 @@ describe('Attendance reason statistics', () => {
           comment: 'Cleaner',
           suspended: true,
         },
-        {
-          bookingId: 1133344,
-          offenderNo: 'G8977UK',
-          eventId: 5,
-          cellLocation: 'LEI-4',
-          startTime: '2019-10-10T14:00:00',
-          timeSlot: 'AM',
-          firstName: 'Offender',
-          lastName: 'Four',
-          comment: 'Cleaner',
-          suspended: false,
```

Comment:
this was previously being filtered in the typescript, but no need now as Prison API only returns the suspended ones